### ISSUE TITLE
Infinite catagories

### ISSUE THREAD

**Issue by yojimble** [PlebeianApp/plebeian.market#417]
of course cats need their own category, but user submitted categories will get crazy? This section feels more like a tag cloud (which is also good, especially where is says the number of products under each heading), but a more compact, maybe tiered, option would be easier for navigating. Is this something that could be set by the person hosting a plebeian instance?

![Opera Snapshot_2024-11-21_132720_staging plebeian market](https://github.com/user-attachments/assets/218b0c4a-bf65-427f-8e50-0894fcc8bc82)

this was 16 pages long a one point I recall!

**Comment by gzuuus** [comment]
Yes, this is a challenge. Users can define their own categories, and these should be displayed. As you can see, the categories are currently sorted by the number of products they contain. One possible solution is to show only the top 4 most popular categories, which would help avoid clutter. However, if the owner of the PM instance is responsible for deciding which categories to display, this could be limiting and may not be the best approach. A better solution might be to allow the instance owner to recommend categories that appear as suggestions when a user wants to set a category

**Comment by yojimble** [comment]
Category suggestions might help, then a user can choose for example if they want to orphan their item by creating a 'bag' category if a 'bagS' category already existed with several items within in. 

There is a lot of trust here for users not to abuse the system to get their items shown above others by spamming the categories
